src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_virtual_network.py

```diff
@@ -213,7 +213,7 @@ def create_vnet(cmd, servername, location, resource_group_name, delegation_servi
                                                    VirtualNetwork(name=vnet_name, location=location,
                                                                   address_space=AddressSpace(
                                                                       address_prefixes=[vnet_address_prefix])))
-    delegation = Delegation(name=delegation_service_name, serv_name=delegation_service_name)
+    delegation = Delegation(name=delegation_service_name, service_name=delegation_service_name)
```

Comment:
We have fix this in #16381, after the horfix is GA, we will merge this change into `dev` branch. So there's no need to modify it again here.

Reply:
Will remove it before check in. Had to include it in the PR for the CI to pass.

Reply:
The hotfix is released. You can merge `dev` branch now😆

Reply:
@evelyn-ys : Reverted the change and merged with dev. Can you take a look and approve/merge this PR whenever you get a chance ?